index.html

```diff
@@ -3676,6 +3678,31 @@ <h2 id="DelayNode">
         </dl>
       </section>
       <section>
+        <h2>
+          DelayOptions
+        </h2>
+        <p>
+          This specifies options for constructing a
+          <a><code>DelayNode</code></a>. All members are optional; if not
+          given, the node is constructed using the normal defaults.
+        </p>
+        <dl title="dictionary DelayOptions : AudioNodeChannelOptions" class=
+        "idl">
+          <dt>
+            double maxDelayTime
+          </dt>
+          <dd>
+            The maximum delay time for the node.
```

Comment:
This is an optional parameter with a default of 1.0 in the existing `createDelay()` method in AudioContext. The default should be specified here.